reference/tax_provider.yml

```diff
@@ -682,6 +687,22 @@ components:
         - quantity
         - type
       title: ItemRequest
+    request-item-tax-property:
+      type: object
+      description: A simple key value pairing allowing merchants to provide an additional input into a tax provider's tax calculation.
+      properties:
+        name:
```

Comment:
After some further discussion we want to shift this to `code` to clarify it's purpose a little more.